doc/source/cheatsheet/cheat_sheet.qmd

```diff
@@ -0,0 +1,239 @@
+---
+title: PyAnsys Units cheat sheet
+format: cheat_sheet-pdf
+version: 0.1.0
```

Comment:
```suggestion
params:
  version: main
```
please add version as pararms, it will automatically align with library version. example https://github.com/ansys/pyansys-geometry/blob/main/doc/source/cheatsheet/cheat_sheet.qmd#L4C1-L5C16